docs/releases/v5_0_0_3357-x86_64.md

```diff
@@ -0,0 +1,29 @@
+# libhkl-v5.0.0.3357-x86_64.tar.gz
+
+[v1.2.0](https://github.com/bluesky/libhkl/releases/tag/v1.2.0) | `libhkl-v5.0.0.3357-x86_64.tar.gz`
+
+NOTE: Cannot build conda-forge package since gsl=2.7.1 not available.
```

Comment:
My guess is that people read only one of these release notes, so probably good to copy this message into the others that are similarly affected.

Reply:
This NOTE was applicable at the time of the v1.2.0 release.  No guarantees this is general.  Let's not copy this around.